### PR TITLE
Tickets/dm 21756

### PR DESCRIPTION
**Pull request by couger01**

### CODE REVIEW

File: config/Electrometer/v1/_labels.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+# there are no standard Electrometer configurations yet
```
Comment (tribeiro) [inline config/Electrometer/v1/_labels.yaml]: Add new line at the end of the file....

File: python/lsst/ts/electrometer/__init__.py [hunk 2]
```diff
@@ -20,3 +20,6 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 from .version import *  # Generated by sconsUtils
+from . import csc
+from . import controller
+from . import model
```
Comment (tribeiro) [inline python/lsst/ts/electrometer/__init__.py]: New line at the end of the file

File: python/lsst/ts/electrometer/controller.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import enum
+
+
+class UnitMode(enum.IntEnum):
```
Comment (tribeiro) [inline python/lsst/ts/electrometer/controller.py]: Are these enums from the CSC interface? if so, they need to go to `ts_idl`

File: python/lsst/ts/electrometer/csc.py [hunk 4]
```diff
@@ -0,0 +1,119 @@
+from lsst.ts import salobj
+import enum
+import pathlib
+from . import model
+
+
+class DetailedState(enum.IntEnum):
```
Comment (tribeiro) [inline python/lsst/ts/electrometer/csc.py]: This need to go to `ts_idl`

File: python/lsst/ts/electrometer/csc.py [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    OFFLINESTATE = 4
+    STANDBYSTATE = 5
+    NOTREADINGSTATE = 6
+    CONFIGURINGSTATE = 7
+    MANUALREADINGSTATE = 8
+    READINGBUFFERSTATE = 9
+    SETDURATIONREADINGSTATE = 10
+
+
+class Units(enum.IntEnum):
```
Comment (tribeiro) [inline python/lsst/ts/electrometer/csc.py]: Probably for `ts_idl`?

File: python/lsst/ts/electrometer/controller.py (hunk 3 above)
Comment (couger01) [reply python/lsst/ts/electrometer/controller.py]: No, the enums are for the controller specifically. The enums in csc need to go into ts_idl, I guess.

File: python/lsst/ts/electrometer/csc.py [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+        self.assert_enabled("stopScan")
+        self.assert_substate(substates=[DetailedState.MANUALREADINGSTATE,
+                                        DetailedState.SETDURATIONREADINGSTATE], action="stopScan")
+        self.detailed_state = DetailedState.READINGBUFFERSTATE
+        self.model.read_buffer()
+        self.detailed_state = DetailedState.NOTREADINGSTATE
+
+    @staticmethod
+    def get_config_pkg():
+        return "ts_config_electrometer"
```
Comment (tribeiro) [inline python/lsst/ts/electrometer/csc.py]: Use `ts_config_ocs`

File: schema/Electrometer.yaml [hunk 7]
```diff
@@ -0,0 +1,83 @@
+$schema: http://json-schema.org/draft-07/schema#
+$id: https://github.com/lsst-ts/ts_electrometer2/blob/master/schema/Electrometer.yaml
+title: Electrometer v1 
+description: Schema for Electrometer configuration files
+type: object
+properties:
+  fits_files_path:
+    description: The path of where the copied fits file are located.
+    type: string
+    default: "/home/saluser/ts_electrometer2/electrometerFitsFiles"
```
Comment (tribeiro) [inline schema/Electrometer.yaml]: Use "~/electrometerFitsFiles"

File: schema/Electrometer.yaml [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+  ip_address:
+    description: The ip address of the file transfer server
+    type: string
+    format: ipv4
+    default: "10.0.100.133"
```
Comment (tribeiro) [inline schema/Electrometer.yaml]: maybe default should be `localhost` or `0.0.0.0`?

File: schema/Electrometer.yaml [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+  range:
+    description: Something
+    type: number
+    default: -0.01
+  integration_time:
+    description: The time that the electrometer opens per scan.
+    type: number
+    default: 0.001
+  median_filter_active:
+    description: Is the the median filter active?
```
Comment (tribeiro) [inline schema/Electrometer.yaml]: Double "the"

**Pull request merged**